roles/setup_assisted_installer/tasks/main.yml

```diff
@@ -40,14 +40,23 @@
     - name: Open ports zone internal and public, for firewalld
       firewalld:
         port: "{{ item.1 }}/tcp"
-        permanent: yes
-        immediate: yes
+        permanent: true
+        immediate: true
         state: enabled
         zone: "{{ item.0 }}"
       loop: "{{ ['internal', 'public'] | product(['8000', port, assisted_service_gui_port, assisted_service_image_service_port]) | list }}"
 
-    - name: Create directories for assisted-installer
+    - name: Delete previous assisted-installer directories
```

Comment:
Can you add a flag for disabling this there maybe some people who wish to keep some of there data around between installs.